config/webhook/manifests.yaml

```diff
@@ -25,7 +25,7 @@ webhooks:
     - CREATE
     - UPDATE
     resources:
-    - openstackcluster
+    - openstackclusters
```

Comment:
I assume this change is for consistency because all other resources are referenced by their plural?

Reply:
we must set it , otherwise the webhook doesn't work at all..

I guess the reason is the resource we created in CRD is plural but not 100% sure..

Reply:
That's some strange voodoo: I thought these were equivalent. It's more consistent like this anyway, so 🤷‍♂️ That can remain a mystery for another time.